luda/editor/client/src/timeline/clip/SubtitleClipComponent.ts

```diff
@@ -51,7 +51,7 @@ export const SubtitleClipComponent: Render<
   const color =
     timelineState.selectedClip?.id === clip.id
       ? ColorUtil.Green
```

Comment:
I think we don't have to change the color of selected one. Would you highlight border only?

Reply:
Because may there are other green clips.